### DISCUSSION TITLE
Is there any way to install version 2.5.9 32-bit?

### DISCUSSION THREAD

**Discussion by Ciccotti** [jacobalberty/firebird-docker#98] [General]
If the answer is yes, what changes do I need to make?

**Comment by mrotteveel** [comment]
I believe the answer is no. Why do you care if it is 32-bit or 64-bit?

**Reply by Ciccotti** [reply]
I use an old database, with 32-bit DLLs

**Reply by jacobalberty** [reply]
@Ciccotti so to be clear you're looking for 32 bit x86 right?

**Reply by arvanus** [reply]
Do you know that the bitness of the client is totally unrelated with the server?
You could have an 64bits server working perfectly with 32bit clients as long as the clients are using the correct 32bits dll

**Reply by mrotteveel** [reply]
@arvanus  I think the OP is referring to UDF DLLs, which must match the bitness of the server.

**Reply by mrotteveel** [reply]
Though that wouldn't work either, because the Firebird Docker images are Linux, not Windows.
